supabase/tools/createStorage.ts

```diff
@@ -23,21 +23,24 @@ async function createPublicBucket(supabase: SupabaseClient, name: string): Promi
     });
 
     if (error) {
-        throw new Error(`Failed to create bucket called ${name}, error: ${error}`);
+        throw new Error(`Failed to create bucket called ${name}, error: ${error.message}`);
     }
 }
 
-async function uploadCongestionZoneFile(supabase: SupabaseClient): Promise<void> {
+async function uploadCongestionZoneFile(
+    supabase: SupabaseClient,
+    bucketName: string
+): Promise<void> {
     const file = fs.readFileSync("supabase/tools/congestion-zone-postcodes.txt");
 
     const { error } = await supabase.storage
-        .from("public")
+        .from(bucketName)
```

Comment:
Is it different for creating buckets and the "from" part needs to be the actual bucket name?

Reply:
this is inconsistent with how we access the bucket in the edge function but this is in fact the syntax that works when i tested locally

Reply:
which is a big shame